is_prime/rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.81.0"
+components = ["llvm-tools", "rustc-dev"]
```

Comment:
Please end your text files with a newline.

You can set up your editor to do that automatically for you.